css/components/modal/_modal-body.scss

```diff
@@ -0,0 +1,159 @@
+.modal-body {
+  display: inline-block;
+  height: auto;
+  overflow-y: scroll;
+  position: relative;
+  width: 100%;
+
+  .wp-editor-area {
+    resize: vertical;
+  }
+
+  .wp-editor-wrap,
+  .wp-editor-container,
+  .wp-editor-container
+  .wp-editor-area {
+    height: 100%;
+  }
+
+  >form {
```

Comment:
Ended up just using your suggestion of a `_form_questions.scss` file - as mentioned in your other comment regarding the modal itself, separating those parts of the form doesn't make sense when they're put on their own.